### PR TITLE
dnsapi for DNSEver

### PR DESCRIPTION
**Pull request by hiskang**

### CODE REVIEW

File: dnsapi/dns_dnsever.sh [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+}
+
+####################  Private functions below ##################################
+
+dnsever_txt(){
+  action="$1"; login_id="$2"; login_password="$3"; fulldomain="$4"; txt="$5"
+
+  _inithttp
+
+  if [ "$_ACME_CURL" ] && [ "${ACME_USE_WGET:-0}" = "0" ]; then
```
Comment (Neilpang) [inline dnsapi/dns_dnsever.sh]: it's too details to do that,  here is how to deal with cookies.
https://github.com/Neilpang/acme.sh/blob/master/dnsapi/dns_cyon.sh#L159

That's to get the cookies headers, and set it back.

You should not know too much about wget or curl.

**Comment by Neilpang** [comment]
Please fix the errors:
https://travis-ci.org/Neilpang/acme.sh/jobs/208711979

**Comment by hiskang** [comment]
I fixed all the Travis CI errors.

File: dnsapi/dns_dnsever.sh [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+####################  Private functions below ##################################
+
+dnsever_txt() {
+  action="$1"
+  login_id="$2"
+  login_password="$3"
+  fulldomain="$4"
+  txt="$5"
+
+  _inithttp
```
Comment (Neilpang) [inline dnsapi/dns_dnsever.sh]: no need to call `_inithttp` here.

File: dnsapi/dns_dnsever.sh [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+  fulldomain="$4"
+  txt="$5"
+
+  response=$(_post "login_id=$login_id&login_password=$login_password" "https://kr.dnsever.com/index.html")
+  if [ $? != 0 ] || [ -z "$response" ]; then
+    _err "dnsever_txt:$action ERROR login failed. Please check https://kr.dnsever.com/index.html with login_id=$login_id login_password=$login_password"
+    return 1
+  fi
+
+  _H2="$(grep PHPSESSID "$HTTP_HEADER" | sed s/^Set-//)"
```
Comment (Neilpang) [inline dnsapi/dns_dnsever.sh]: why use `_H2`  here ?  did you ever use `_H1` ?
Please use _H1.

File: dnsapi/README.md [hunk 4]
```diff
@@ -348,6 +348,20 @@ Ok, let's issue a cert now:
 ```
 acme.sh --issue --dns dns_gandi_livedns -d example.com -d www.example.com
 ```
+## 19. Use DNSEver (https://www.dnsever.com/)
```
Comment (Neilpang) [inline dnsapi/README.md]: your number is not `19` anymore,  please move to the last.

**Comment by Neilpang** [comment]
@hiskang 

Sorry, but can you please rebase to the latest code ?

**Pull request closed without merging**